tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -46,6 +47,8 @@ public void ShowBarWindow()
         }
         else
         {
+            PInvoke.ShowWindow(DBarWindow.CurrentHwnd, SHOW_WINDOW_CMD.SW_RESTORE);
+
             // Activate is unreliable so use SetForegroundWindow
             PInvoke.SetForegroundWindow(DBarWindow.CurrentHwnd);

```

Comment:
Is this still needed now that `ShowWindow` is added?

Reply:
it is still needed as ShowWindow does not transfer foreground rights